homeassistant/components/tractive/sensor.py

```diff
@@ -119,6 +124,19 @@ async def async_added_to_hass(self):
         device_class=DEVICE_CLASS_BATTERY,
         entity_class=TractiveHardwareSensor,
     ),
+    TractiveSensorEntityDescription(
+        key=ATTR_LIVE_TRACKING_REMAINING,
+        name="Live tracking remaining time",
+        icon="mdi:clock-end",
+        native_unit_of_measurement=TIME_SECONDS,
```

Comment:
We shouldn't store countdown times in the state machine. Convert it to a time stamp instead